File: playground_series/s3e14/py/baseline.py
import polars as pl
import statistics

trn = pl.read_csv("./playground_series/s3e14/data/train.csv")
tst = pl.read_csv("./playground_series/s3e14/data/test.csv")

trn.shape

#getting mean yield
pred = statistics.mean(trn['yield'])

#creating submission
sub = pl.DataFrame(
    {
        "id": tst['id'],
        "yield": pred,
    }
)

#write out
sub.write_csv("./playground_series/s3e14/submissions/baseline_py.csv")

File: playground_series/s4e9/baseline.jl
using CSV
using DataFrames
using Statistics

trn = CSV.read("playground_series/s4e9/data/train.csv", DataFrame)
tst = CSV.read("playground_series/s4e9/data/test.csv", DataFrame)

avg_price = mean(trn.:price)

sub = DataFrame(
    id=tst.:id,
    price=avg_price
)

CSV.write("playground_series/s4e9/submissions/baseline.csv", sub)